### DISCUSSION TITLE
Iris 3.5 libnetcdf pin?

### DISCUSSION THREAD

**Discussion by trexfeathers** [SciTools/iris#5213] [Polls]
libnetcdf `v4.9.1` has caused performance regressions (#5182), and produced many new low-level warnings of minimal consequence (#5187). Our two corrective options:

* Investigate fixes before the release - complicated, time-consuming, need to sacrifice other 3.5 work
* Pin `libnetcdf <4.9` - pins are unsustainable long term and create awkwardness for users creating environments.

**Comment by bjlittle** [comment]
@trexfeathers If we vote to temporarily pin we should raise an issue to ensure that we unpin again 👍
